README.md

```diff
@@ -2,46 +2,108 @@
 
 EPFL ENAC-IT Continuous Deployment
 
+## Create two secrets in your repository
+
+- under your repository settings in /settings/secrets/actions
+- Add DEPLOYMENT_SECRET and DEPLOYMENT_ID given by the linux enacit-linux-sysadmins@epf.ch
+
 ## Add Continuous Deployment to PROD hosting
 
 Create `.github/workflows/deploy-prod.yml` containing :
 
 ```yml
-# https://github.com/EPFL-ENAC/github-actions-runner#readme
+# https://github.com/EPFL-ENAC/epfl-enac-deploy-action#readme
 name: deploy-prod
 
-on:
+"on":
   push:
     branches:
       - main
 
 jobs:
   deploy:
-    runs-on: [self-hosted, deploy, prod]
+    runs-on: ubuntu-latest
     steps:
       - uses: EPFL-ENAC/epfl-enac-deploy-action@main
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          ENAC_IT4R_CD_environment: "prod"
+          ENAC_IT4R_CD_deployment_secret: ${{ secrets.DEPLOYMENT_SECRET }}
+          ENAC_IT4R_CD_deployment_id: ${{ secrets.DEPLOYMENT_ID }}
 ```
 
 ## Add Continuous Deployment to TEST hosting
 
 Create `.github/workflows/deploy-test.yml` containing :
 
 ```yml
-# https://github.com/EPFL-ENAC/github-actions-runner#readme
+# https://github.com/EPFL-ENAC/epfl-enac-deploy-action#readme
 name: deploy-test
 
-on:
+"on":
   push:
     branches:
-      - develop
+      - dev
 
 jobs:
   deploy:
-    runs-on: [self-hosted, deploy, test]
+    runs-on: ubuntu-latest
     steps:
       - uses: EPFL-ENAC/epfl-enac-deploy-action@main
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          ENAC_IT4R_CD_environment: "dev"
+          ENAC_IT4R_CD_deployment_secret: ${{ secrets.DEPLOYMENT_SECRET }}
+          ENAC_IT4R_CD_deployment_id: ${{ secrets.DEPLOYMENT_ID }}
+```
+
+## Polling for status with a timeout
+
+This script makes a POST request to a given URL until the status is `finished` or `error`
+Initially created for checking if a enac deployment is complete (via enacit-ansible)
+
+### Inputs
```

Comment:
The following sections should be at the next sub-level ####

Reply:
Thanks! I moved everything!